src/Kale.hs

```diff
@@ -50,6 +50,18 @@ data Task = Task
     }
     deriving (Eq, Show)
 
+type Qualified = String
+type Module = String
```

Comment:
I would prefer to see a `newtype` here -- a type alias doesn't give you any safety, and if it's just aliasing a single thing, it tends to make it more difficult to see what it actually is.